content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -0,0 +1,253 @@
+---
+reviewers:
+- eparis
+- pmorie
+title: कॉन्फ़िगमैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
+content_type: ट्यूटोरियल
+---
+
+<!-- overview -->
+
+यह पृष्ठ कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करने का वास्तविक विश्व उदाहरण प्रदान करता है और [कॉन्फिग मैप का उपयोग करने के लिए पॉड को कॉन्फ़िगर करें] (/docs/tasks/configure-pod-container/configure-pod-configmap/) कार्य पर आधारित है।
+
+
+
+## {{% heading "objectives" %}}
+
+
+* रेडिस कॉन्फ़िगरेशन वैल्यू के साथ एक कॉन्फ़िगमैप बनाएं
+* एक रेडिस पॉड बनाएं जो बनाए गए कॉन्फ़िगमैप को माउंट और उपयोग करता है
+* जांच करें कि कॉन्फ़िगरेशन सही ढंग से लागू किया गया था
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* इस पृष्ठ पर दिखाया गया उदाहरण `kubectl` 1.14 और उससे ऊपर के संस्करण के साथ काम करता है।
+* समझें [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+
+
+
+<!-- lessoncontent -->
+
+
+## वास्तविक विश्व उदाहरण: कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
```

Comment:
Consider incorporating English text for the link to avoid non-English characters in the URL of the section.
```suggestion
## वास्तविक विश्व उदाहरण: कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करना {#real-world-example-configuring-redis-using-a-configmap}
```